bin/ocFunctions.inc

```diff
@@ -1799,6 +1799,75 @@ function getSecret() {
   )
 }
 
+function buildPromptCreds(){
+  namespaceName=${1}
+  newCredName=${2}
+  dockerReg=${3}
+  credEnvs=${4}
+  userName=${5}
+  password=${6}
+  
+  if ([ -z ${userName} ] || [ -z ${password} ]); then
+    echoWarning "Please enter your ${dockerReg} username"
+    read userName
+
+    echoWarning "Please enter your ${dockerReg} password"
```

Comment:
The requests for a "password" is a bit misleading.  Should the user be setting up and using a token rather than a password?

Reply:
@WadeBarnes most docker registries use a password right? I'm not quite sure what you mean by token

Reply:
Docker Hub, and many other services, allow users to setup Access Tokens for authentications so you are not using your actual username and password to access the services.  Access Tokens can be created easily and revoked/deleted in the case of suspected compromise without affecting your actual credentials.

Reply:
@wadeking98, It should.

Reply:
changed wording to ask user for password or token